includes/Core/Admin/Plugin_Action_Links.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Admin\Plugin_Action_Links
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Admin;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Permissions\Permissions;
+
+/**
+ * Class for managing plugin action links.
+ *
+ * @since 1.41.0
+ * @access private
+ * @ignore
+ */
+class Plugin_Action_Links {
+
+	/**
+	 * Plugin context.
+	 *
+	 * @since 1.41.0
+	 * @var Context
+	 */
+	private $context;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 1.41.0
+	 *
+	 * @param Context $context      Plugin context.
+	 */
+	public function __construct(
+		Context $context
+	) {
+		$this->context = $context;
+	}
+
+	/**
+	 * Registers functionality through WordPress hooks.
+	 *
+	 * @since 1.24.0
+	 */
+	public function register() {
+		add_filter(
+			'plugin_action_links_' . GOOGLESITEKIT_PLUGIN_BASENAME,
+			function ( $links ) {
+
+				if ( current_user_can( Permissions::MANAGE_OPTIONS ) ) {
+					$settings_url = $this->context->admin_url( 'settings' );
+
+					$settings_link = '<a href="' . $settings_url . '">' . __( 'Settings', 'default' ) . '</a>';
```

Comment:
A few things here, but most importantly, we always* need to use the `google-site-kit` text domain for our translations. There are a few exceptions to this, but that's the general rule (our ESLint config has better checks for this). While it would probably be safe to use the core translation here, I'm not sure if that is the intention so I have to assume it is not.

In this case, let's use the same as it is defined for the menu which uses our own text domain 👍  https://github.com/google/site-kit-wp/blob/7ff89bbf73b310573a125887b80469941bbce438/includes/Core/Admin/Screens.php#L327

Second, we generally want to use WP escaping functions on any dynamic parts that we output. PHPCS will error if this is not done when outputting HTML, but I think it's not complaining here due to just building an HTML string. Let's use `esc_url` on the URL and for the translatable string, WP has a `esc_html__` version of `__` which escapes any HTML from the translated output (which is just a good practice if we don't expect there to be any).

Finally, it would be a little cleaner to use `sprintf` here to build the string rather than many concatenations but nothing wrong with that. This does make it a bit easier to do something with each part when filling.